_drafts/contribute-to-ansible.markdown

```diff
@@ -9,11 +9,22 @@
     become Ansible contributor
 
 ## Prerequisites
+Contribute to ansible needs some prerequisites.
 
-    un ordinateur avec python (2.7 ou 3.5)
-    pipenv
-    un compte github
-    git installé
+Ansible is written in python so you need to install different versions for
+ensure compatibility:
+- python 2.7
+- python 3.5
+
+For improve development lifecycle we strongly encourage you to use [pipenv](https://github.com/kennethreitz/pipenv).
+Instead of you can use mainstream tools like `virtualenv` and `pip`.
+
+Ansible project is hosted on github so [you must to create an github account](https://github.com/) for deal with us.
```

Comment:
+Ansible project is hosted on github so [you must to create an github account](https://github.com/) to deal with it.